tests/ArraySize/yosys_script

```diff
@@ -1,5 +1,5 @@
 plugin -i uhdm
-read_uhdm -debug top.uhdm
+tcl ../tests/ArraySize/read.tcl
```

Comment:
I don't like that we assumes here that we are in specific folder. I think it would be better to change whole ``yosys_script`` to tcl and remove ``read.tcl`` file. I think it is possible, that in the future we would like to access another environment variables in later steps of the processing.

Reply:
I thought that it is only a temporary solution to test at least one test by the other frontend.

Reply:
The problem with temporary solutions is that they tends to stay permanent and changing whole ``yosys_script`` to tcl is not a big deal.

I know it introduces ``Command name collision: found pre-existing command`` warning, but it is safe and we don't need to worry about it.

Reply:
I am trying to change this now in all tests. There is a problem with for example `proc` command, because there is such command already in tcl. In this case there was no warning, but fortunately there was wrong number of arguments, so it gave error. If the number of argument was correct, the test would probably pass and we wouldn't know about the changed behavior.

Reply:
True, according to yosys help, ``proc`` and ``rename`` commands are overwriten:
```
Yosys commands' proc' and 'rename' are wrapped to tcl commands 'procs' and 'renames'
in order to avoid a name collision with the built in commands
```
In this case, I'm proposing another solution, as content of the ``read.tcl`` will probably be the same for all tests, please generate it in build directory using Makefile. This way we won't need to point to the specific test folder.

Reply:
Ok. I think it's much better idea.